src/main/java/com/onlinebookstore/dto/book/UpdateBookDto.java

```diff
@@ -0,0 +1,32 @@
+package com.onlinebookstore.dto.book;
+
+import com.onlinebookstore.model.Category;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.math.BigDecimal;
+import java.util.Set;
+import lombok.Data;
+
+@Data
+public class UpdateBookDto {
+    @NotBlank
+    @Size(min = 1, max = 100)
+    private String title;
+    @NotBlank
+    @Size(min = 1, max = 50)
+    private String author;
+    @NotBlank
+    @Size(min = 10, max = 13)
+    private String isbn;
+    @NotNull
+    @Min(0)
+    private BigDecimal price;
+    @NotBlank
+    @Size(min = 6, max = 500)
+    private String description;
+    @NotBlank
+    private String coverImage;
+    private Set<Category> categories;
```

Comment:
Entity should not be a part of request/response DTO. Don't forget to change `BookMapper`
```suggestion
    private Set<Long> categoryIds;
```